indexing-service/src/main/java/org/apache/druid/indexing/common/actions/CheckPointDataSourceMetadataAction.java

```diff
@@ -75,6 +76,28 @@ public SeekableStreamDataSourceMetadata getPreviousCheckPoint()
     return checkpointMetadata;
   }
 
+  // For backwards compatibility
+  @Deprecated
+  @JsonProperty
+  public SeekableStreamDataSourceMetadata getCurrentCheckPoint()
```

Comment:
Since this is just for Jackson, you can make it private and then get rid of the `@Deprecated` if you want